GIFrameworkMaps.Web/Views/ManagementAnalytics/Index.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@model GIFrameworkMaps.Data.Models.ViewModels.Management.AnalyticsViewModel
+@{
+    ViewData["Title"] = "System management - analytics";
+}
+
+<h1>@ViewData["Title"]</h1>
+
+<p class="lead">
+    Add, remove and manage analytics tracking and cookie controls.
+</p>
+
+<p>
+    To help with understanding how users are interacting with this site you may want to add a variety of analytic tools. 
+    These often capture information about which pages are visited and how users interact with the application.
+</p>
+<p>
+    There may also be cookie control tools but we have assumed these will be packaged within your analytic tool choice. 
+</p>
```

Comment:
I'm not sure we need the line of text on line 17, it doesn't really add anything at this point.

Mention of cookie controls might be more appropriate on the create/edit view pages where the option is given to pick one.